### ISSUE TITLE
Do we need stories? 

### ISSUE THREAD

**Issue by JosephMcArthur** [OAButton/discussion#638]
Stories impose a significant burden on our team & our users. Currently, they need moderation and relatively careful crafting by users.

It's worth understanding two questions
1. Are stories written by users significantly more effective than nothing?
2. Are stories written by users significantly more effective than stories written by our team, and merely selected by users (probably at sign up) more effective? 

If the answer to any of these questions is no, there is a strong case to remove stories. To answer this question we need to do:
- [ ] A / B testing on 1 - @natalianorori 
- [ ] Craft some examples of "generic" stories - Me
- [ ] Understand authors opinions on this & if qualitatively our stories perform better than ones provided by users - @svmelton

**Comment by JosephMcArthur** [comment]
Note to self, if we break down story rating into good / bad thats a lot less mental effort than our current scale. This can then also be streamlined into something else (like a button to send to authors)

**Comment by natalianorori** [comment]
yes, but these ratings would be subjective to an individual's personal
opinion, disregarding the type of scale being used. I think we could
discuss the definition of a "good story" as a team, and come up with a good
guide (as the one opencon reviewers get) :)

**Comment by JosephMcArthur** [comment]
@svmelton can you summarise your finding from chatting with authors here please?

**Comment by natalianorori** [comment]
@svmelton I am running an AB test to find out if the response rate and time drop if we don't add the user's name and the story. I am tracking the data here https://docs.google.com/spreadsheets/d/1XHVvLMUM3UT7vggIQwhWVyqlnIE37MltrPfmrgzxUcU/edit#gid=0 , It would be great to get a summary of authors opinions you've gotten from user testing

**Comment by svmelton** [comment]
@natalianorori Cool! All the authors I've spoken to said that story was important to them—much more important than name, and slightly more important than profession.

**Comment by JosephMcArthur** [comment]
@natalianorori if you have any results here can you comment them here?

**Comment by natalianorori** [comment]
5% response rate for requests with stories and 0% response rate for requests with no stories. 

This was a small test, we compared last 100 moderated requests in the system and sent 100 without stories to YAMM. I could expand it if you want a relevant sample size, but I have a hunch that we already now the answer: stories matter.

**Comment by natalianorori** [comment]
relevant for #936

**Comment by JosephMcArthur** [comment]
So, I think the results of that test are kind of clear and gels with everything we think. I think we need to try and keep stories, i.e personalized reasons why the person should deposit, but we need to think about how to lower the burden substantially for users and us. 

Big ways we can do that include:
* #936
* #428 / #412 which was shut originally but I think we should reconsider as it means we don't need to moderate
* #764
* be clearer about instructions. I'm seeing some people put in their names, the paper title etc. Maybe something responsive would be good e.g if "dear" is seen, display X. If doi is seen, say Y. 
* if a user does X good stories, automatically validate after that.
* have a reporting function for issue with stories (this can let us a be a little more hands off as issues can be caught by others)
* other ways I've not thought of yet. 

@natalianorori you said volunteers liked doing stories, yes? Perhaps we specialize something there (e. g simple page where they can be rated or use mturk again).
